library/src/main/res/layout/rxw_list_view_card.xml

```diff
@@ -5,19 +5,20 @@
     android:layout_height="wrap_content"
     android:layout_width="match_parent">
 
-    <LinearLayout
+    <RelativeLayout
         android:layout_height="wrap_content"
-        android:layout_width="match_parent"
-        android:orientation="vertical">
+        android:layout_width="match_parent">
 
-        <LinearLayout
-            android:id="@+id/container"
-            android:layout_height="wrap_content"
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/recycler_view"
             android:layout_width="match_parent"
-            android:orientation="vertical"/>
+            android:layout_height="wrap_content"
+            android:overScrollMode="never"
+            android:nestedScrollingEnabled="false"/>
```

Comment:
I don't know that this does what we need it to do all the time. I've read that this property doesn't always work.